crates/xilem_web/src/view.rs

```diff
@@ -90,6 +90,8 @@ xilem_core::generate_anyview_trait! {AnyView, View, ViewMarker, Cx, ChangeFlags,
 xilem_core::generate_memoize_view! {Memoize, MemoizeState, View, ViewMarker, Cx, ChangeFlags, static_view, memoize;}
 xilem_core::generate_adapt_view! {View, Cx, ChangeFlags;}
 xilem_core::generate_adapt_state_view! {View, Cx, ChangeFlags;}
+xilem_core::generate_rc_view!((std::rc::Rc), View, ViewMarker, Cx, ChangeFlags;);
+xilem_core::generate_rc_view!((std::sync::Arc), View, ViewMarker, Cx, ChangeFlags; + Sync);
```

Comment:
Seems unnecessarily inconsistent that this macro is invoked using round parentheses while the others are invoked using curly braces.

Reply:
Whoops you're right, was a little bit too fast there.